README.md

```diff
@@ -126,7 +167,7 @@ yarn install
 yarn local
 ```
 
-## Lighthouse Score Visualizations (Labs feature)
+## Preview Lighthouse results within the Netlify UI
```

Comment:
I changed to this reflect the new verb-oriented headers. Let me know if it makes more sense to keep the noun in the header/somewhere in this section more prominently for Search. cc @MetalMatt-Netlify